stack/base/before-notebook.d/40_prepare-aiida.sh

```diff
@@ -48,7 +48,8 @@ if [[ ${NEED_SETUP_PROFILE} == true ]]; then
       exit 1
     fi
 
-    verdi computer show ${computer_name} || verdi computer setup \
+    # TODO: Load config from YAML file
```

Comment:
This is not urgent and didn't bring speedup I assume? Maybe using verdi is more straightforward.

Reply:
No, this wouldn't bring any speed up since you would still need to invoke verdi. It would be a bit cleaner since the computer would be statically defined in a yaml file, and not via cmdline parameters, but it's not important for this PR.

Reply:
On second thought, since the number of CPUs is defined dynamically, I think we can't have config in yaml. I've removed the TODO.